internal/providers/google_chat/google_chat.go

```diff
@@ -69,7 +70,7 @@ func NewGoogleChat(opts GoogleChatOpts) (*GoogleChatManager, error) {
 	}
 
 	// Load the template.
-	tmpl, err := template.New("message.tmpl").Funcs(templateFuncMap).ParseFiles(opts.Template)
+	tmpl, err := template.New(filepath.Base(opts.Template)).Funcs(templateFuncMap).ParseFiles(opts.Template)
```

Comment:
Where does `opts.Template` come from?

Reply:
it is GoogleChatOpts field with path to template, You already used it in the right part (ParseFiles method)